lang/syn/src/codegen/program/entry.rs

```diff
@@ -51,7 +51,7 @@ pub fn generate(program: &Program) -> proc_macro2::TokenStream {
         ///
         /// The `entry` function here, defines the standard entry to a Solana
         /// program, where execution begins.
-        #[cfg(not(feature = "no-entrypoint"))]
+        #[cfg(any(not(feature = "no-entrypoint"), feature = "test-entrypoint"))]
```

Comment:
Happy to merge as is, but what do you think about simply removing this feature flag all together, i.e., always having the function exposed?

Reply:
I suppose it might be better to just do that, since don't really see any harm in making it public by default. It's pretty useful for testing, but also could be useful as a cheaper way of simulating an action without having to send to an RPC node.